src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -271,8 +271,12 @@ private String getQuestionResultsStatisticsHtmlQuestionView(List<FeedbackRespons
         for(Map.Entry<String, StudentResultSummary> entry : studentResults.entrySet()){
             StudentResultSummary summary = entry.getValue();
             String email = entry.getKey();
-            String name = bundle.roster.getStudentForEmail(email).name;
-            String team = bundle.roster.getStudentForEmail(email).team;
+            StudentAttributes student = bundle.roster.getStudentForEmail(email);
+            if (!bundle.sectionTeamNameTable.containsKey(student.section)) {
```

Comment:
This check shouldn't be necessary as `studentResults` only contains entries for students in teams which are in `sectionTeamNameTable`. This is because `getTeamNames` in line 233 already gets the team names from `sectionTeamNameTable`.